.env

```diff
@@ -4,5 +4,6 @@ export SECRET_KEY_BASE=aBi07mDCHwwdqkunV/mx7LlDilXqy7GeZlrXMOm1Qxtdx9e6KfZ2BsF7O
 export TARGET_EMAIL=jackacarlisle@gmail.com
 export SES_SERVER=email-smtp.eu-west-1.amazonaws.com
 export SES_PORT=25
-export SMTP_USERNAME=AKIAIUWYA3RHAHIAP62Q
-export SMTP_PASSWORD=Apv5OSMNgaqncYUlfl2VLnIsklyULMBI9i50VNEhEWA7
+export SMTP_USERNAME=AKIAI7P76NEEEBT6QGKQ
```

Comment:
@jackcarlisle leaving _REAL_ AWS Keys in project is _unacceptable_ you _know_ better. 
Please `git rm .env` _emmediately_, reset the keys on AWS 
and _please_ don't let this happen again. 

@markwilliamfirth `this` is why we _don't_ "hand out" the @dwyl AWS keys (_to the kingdom_!!) without _appropriate **safeguards**_!
relates to: https://github.com/dwyl/hq/issues/221#issuecomment-302083828